owncloudApp/src/main/java/com/owncloud/android/presentation/ui/files/filelist/MainFileListViewModel.kt

```diff
@@ -25,18 +25,19 @@ import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.owncloud.android.R
-import com.owncloud.android.domain.UseCaseResult
 import com.owncloud.android.domain.files.model.OCFile
 import com.owncloud.android.domain.files.usecases.GetFolderContentUseCase
 import com.owncloud.android.domain.utils.Event
 import com.owncloud.android.presentation.UIResult
+import com.owncloud.android.presentation.toUIResult
 import com.owncloud.android.providers.ContextProvider
-import kotlinx.coroutines.Dispatchers
+import com.owncloud.android.providers.CoroutinesDispatcherProvider
 import kotlinx.coroutines.launch
 
 class MainFileListViewModel(
-    private val getFolderContentUseCase: GetFolderContentUseCase,
+    private val getFolderContentAsFlowUseCase: GetFolderContentUseCase,
```

Comment:
Not flow as far as I see, let's rename it.

By the way, I see that we keep using the `GetFolderContentUseCase`, shouldn't we use `GetFolderContentAsLiveDataUseCase` instead?

Reply:
Oh, I forgot to change it after some testing. Thank you.